bin/cert-generator-wrapper.sh.in

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+
+usage() {
+	cat <<EOF
+$(basename "${0}"): SCF Certificate Generator
+
+  -d <domain>:     Sets the top level domain for the cluster
+  -h:              Displays this help message
+  -n <namespace>:  Sets namespace, default: 'cf'
+  -o <output dir>: Sets output directory, default: \`pwd\`
```

Comment:
Should possibly mention that the script assumes that out_dir exists and will not create it. Consequence of lines 33ff